hazelcast/src/main/java/com/hazelcast/spi/OperationAccessor.java

```diff
@@ -43,6 +44,11 @@ public static boolean isMigrationOperation(Operation op) {
                 && op.getClass().getClassLoader() == THIS_CLASS_LOADER;
     }
 
+    public static boolean isWanReplicationOperation(Operation op) {
+        return op instanceof WanReplicationOperation
+                && op.getClass().getClassLoader() == THIS_CLASS_LOADER;
```

Comment:
Ok. I see that already is being done on the OperationAccessor by other functions. Not very nice. Can you move all of them to a utility class called e..g Operations?